scripts/index.js

```diff
@@ -0,0 +1,103 @@
+let currentTemp = 65;
+
+const updateTempColor = currentTemp => {
+    const tempContainer = document.getElementById("currentTemp");
+    let color = "green";
+    if (currentTemp >= 80) {
+        color = "red";
+    } else if (currentTemp >= 70) {
+        color = "orange";
+    } else if (currentTemp >= 60) {
+        color = "yellow";
+    } else if (currentTemp >= 50) {
+        color = "green";
+    } else if (currentTemp < 50) {
+        color = "teal";
+    }
+    tempContainer.classList = color;
+};
+
+const updateTemp = currentTemp => {
+    const tempContainer = document.getElementById("currentTemp");
+    tempContainer.textContent = currentTemp;
+    updateTempColor(currentTemp);
+    updateLandscape(currentTemp);
+};
+
+const increaseTemp = () => {
+    currentTemp += 1;
+    updateTemp(currentTemp);
+};
+
+const decreaseTemp = () => {
+    currentTemp -= 1;
+    updateTemp(currentTemp);
+};
```

Comment:
Just a note that you can combine these functions with:

```suggestion
const changeTemp = (increment) => {
    currentTemp += increment;
    updateTemp(currentTemp);
};
```

Then we can register the event handler with an anonymous function like this:

```javascript
increaseTempButton.addEventListener("click", () => updateTemp( +1 ) );
```